server/src/main/java/model/placement/field/OccupiedField.java

```diff
@@ -7,8 +7,8 @@
 
     /**
      * Simply returns certain FieldState
-     * @return
-     * return parameter is of FieldState and always is OCCUPIED which
+     *
+     * @return return parameter is of FieldState and always is OCCUPIED which
```

Comment:
return return
Just reminder, because I saw it a lot :)

Reply:
Thanks for noticing. Done.